ert3/console/_console.py

```diff
@@ -78,6 +96,44 @@ def _build_argparser():
     return parser
 
 
+def _get_ert3_examples_path():
+    pkg_examples_path = pathlib.Path(pkg.resource_filename("ert3_examples", ""))
+    # check that examples folder exist
+    if not pkg_examples_path.exists():
+        raise ert3.exceptions.IllegalWorkspaceOperation(
```

Comment:
This seems to be an internal error...? Perhaps it should not be an `ErtError`, but present itself as a stack trace and a bug?

Reply:
Also, adding `pkg_examples_path` to the error message will help debugging...